napari/_qt/utils.py

```diff
@@ -127,9 +130,24 @@ def qt_signals_blocked(obj):
 
 
 def disable_with_opacity(obj, widget_list, disabled):
+    """Set enabled state on a list of widgets. If disabled, decrease opacity"""
     for wdg in widget_list:
         widget = getattr(obj, wdg)
         widget.setEnabled(obj.layer.editable)
         op = QGraphicsOpacityEffect(obj)
         op.setOpacity(1 if obj.layer.editable else 0.5)
         widget.setGraphicsEffect(op)
+
+
+@lru_cache(maxsize=64)
+def square_pixmap(size):
+    """Create a white/black hollow square pixmap. For use as labels cursor."""
```

Comment:
The current square is actually white / black / white which i found gave better contrast over many images - did you intentionally make this one just white / black? (It may be fine, just curious if it was done on purpose)

Reply:
Having spent more time with it, the new cursor with just the white / black is really quite nice, I like that the thickness of the border remains the same with size, something not possible with the old approach. One thing though is you need to add some clipping to size. In the old version I had `size < 10 or size > 300` and it transition to the plus symbol. This is needed if you are really zoomed out and make the brush very small, then now you can get this error message `WARNING: QPainter::setPen: Painter not active` and see nothing, or if you are very zoomed and make brush very large then the square becomes so big that it is larger than the whole screen and you can't see it anymore. Both those behaviours are undesirable

Reply:
even better, current behavior works great - thanks!!